packages/editor/src/components/inserter-sidebar/style.scss

```diff
@@ -7,16 +7,18 @@
 }
 
 .editor-inserter-sidebar__header {
-	padding-top: $grid-unit-10;
-	padding-right: $grid-unit-10;
+	position: relative;
 	display: flex;
 	justify-content: flex-end;
+	z-index: 2;
+}
+
+.editor-inserter-sidebar__close-button {
+	position: absolute;
+	top: $grid-unit-15;
+	right: $grid-unit-15;
 }
 
 .editor-inserter-sidebar__content {
-	// Leave space for the close button
-	height: calc(100% - #{$button-size} - #{$grid-unit-10});
```

Comment:
No longer necessary because the close button can sit next to the tabs rather than above them.